### PR TITLE
Handled Issue #137

### PR DESCRIPTION
**Pull request by vipulsnp**
This PR solves issue #137 

**What does this PR do?**

**Any background context you want to provide?**

**Screenshots?**

### CODE REVIEW

**Comment by vipulsnp** [comment]
Also handled `No white spaces` at beginning at end.

File: src/FormValidation.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import { checkUnique } from './firestore/profileSettings';
 
+export function noWhiteSpaceAtBeginEnd(data) {
```
Comment (irshadjsr21) [inline src/FormValidation.js:3]: Instead of checking for whitespaces and beginning and end, we can just remove them before adding to firebase.

Comment (vipulsnp) [reply src/FormValidation.js:3]: But it will create errors if someone just enters a single space in required fields.

Comment (vipulsnp) [reply src/FormValidation.js]: Removed validation from this pr

**Pull request merged**